### PR TITLE
Add TXT record support

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

**Review by kuritka — APPROVED** [review]
the most of the code was moved from `kubernetes.go` to `endpoints.go`.

`var addrs []net.IP` has been changed to `var addrs []string` with all consequences. 

TXT support added 

LGTM 👍

File: endpoints.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		geoData := &geo{}
+		log.Infof("processing IP %+v", ip)
+		err = db.Lookup(net.ParseIP(ip), geoData)
+		if err != nil {
+			log.Error(err)
+			continue
+		}
+
+		log.Infof("IP info: %+v", geoData.DC)
+		if clientGeo.DC == geoData.DC && geoData.DC != "" {
```
Comment (jkremser) [inline endpoints.go:56]: nit: `geoData.DC != ""` condition is redundant here, because we know that `clientGeo.DC` is not `""` (line 39)

**Review by jkremser — APPROVED** [review]

File: endpoints.go (hunk 1 above)
Comment (k0da) [reply endpoints.go]: @jkremser right, amended.

**Review by somaritane — APPROVED** [review]
lgtm

**Pull request merged**